app/ruffle/src/main/java/rs/ruffle/FullscreenNativeActivity.java

```diff
@@ -0,0 +1,19 @@
+package rs.ruffle;
+
+import android.app.NativeActivity;
+import android.os.Bundle;
+import android.view.Window;
+import android.view.WindowManager;
+
+public class FullscreenNativeActivity extends NativeActivity {
```

Comment:
Oh wow, this can be done, and this easily, and works this well! It didn't even occur to me...

This is probably gonna make a few things I've been experimenting with lately so much easier!

Reply:
I was thinking it might be possible to get a view on top of the native view through here, but I'm still experimenting with that (and gamepad input too potentially)

Reply:
Ah, that too would be really cool, if at all possible. I was going to pull in `egui` to do the context menu and the button overlay stuff from Rust, but if it can be done from Java this way, that might actually be easier and more convenient!